README.md

```diff
@@ -55,6 +55,17 @@ $ yarn start
 Once the test server has started, you can browse a local demo, and run tests in
 a browser by visiting `http://localhost:8080/`.
 
+##### Validate Licensing
+
+[Apache Rat (Release Audit Tool)](https://creadur.apache.org/rat/) is a
+preferred code license checking tool used by [the ASF](https://apache.org/).
+The included `.ratignore` file contains a list of files to exclude from scans.
+
+To check for included licenses, run the following and view the output report:
+```sh
+java -jar ~/bin/apache-rat-0.13/apache-rat-0.13.jar -E .ratignore -d . > rat_report.txt
+```
+
```

Comment:
What do you think about moving this later in the README? Some users may care, but I would guess most would not need to see this first.

Reply:
Perhaps as a subheading under our License section, below.